go.mod

```diff
@@ -4,24 +4,25 @@ go 1.20
 
 require (
 	github.com/filecoin-project/go-bitfield v0.2.4
+	github.com/ipfs/go-cid v0.0.6
```

Comment:
Rant: It's annoying that cbor-gen forces this dependency even though we have no CIDs